data/config/brh.json

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
Yeah, that's a good point. It made sense to have this in GH while we were working on feature branches, but we probably don't want to merge this in to master

Reply:
Although, I do see we have a precedent for other commons to have their config here https://github.com/uc-cdis/data-portal/tree/master/data/config

I'll try to figure out if this is appropriate or not. I feel like we had this convo before at some point, but I can't remember the outcome

Reply:
it's true that the data/config JSON files in data-portal are often duplicates of configuration data from gitops-qa. but i find them convenient for local development -- to test the ecosystem browser locally, it's helpful to have a config file so that you can set `APP=brh` following the precedent set by other commons ~